### PR TITLE
Elaborate the release instruction

### PR DESCRIPTION
**Pull request by style95**
This is to add more information to release the `openwhisk` repository.

### CODE REVIEW

File: docs/release_instructions.md [hunk 1]
```diff
@@ -132,13 +167,18 @@ provide historical documentation of project releases.
...
+**Important Note1:** If you are releasing a Node.js package which
 contains a `package.json`, you should make sure the `version`
 specified in `package.json` matches your release version. If you
 also have a `package-lock.json` file, it too must have a matching
 `version` property. See additional tips for [releasing npm
 packages](#publishing-to-npm).
 
+**Important Note2:** If you are releasing the `openwhisk` or `openwhisk-deploy-kube` repositories, 
+the branch and commit hash in the release contents should be the release branch and the latest commit hash in the release branch.
+Please be especially noted that they are not from the `master` branch.
```
Comment (rabbah) [inline docs/release_instructions.md:179]:
```suggestion
Please be careful not to use the `master` branch.
```

File: docs/release_instructions.md [hunk 2]
```diff
@@ -132,13 +167,18 @@ provide historical documentation of project releases.
      - repository: URL of the repository
      - branch: git branch being released (`master` or a release branch name)
 
-**Important Note:** If you are releasing a Node.js package which
+**Important Note1:** If you are releasing a Node.js package which
```
Comment (rabbah) [inline docs/release_instructions.md:170]:
```suggestion
**Important Note 1:** If you are releasing a Node.js package which
```

File: docs/release_instructions.md [hunk 3]
```diff
@@ -132,13 +167,18 @@ provide historical documentation of project releases.
...
 contains a `package.json`, you should make sure the `version`
 specified in `package.json` matches your release version. If you
 also have a `package-lock.json` file, it too must have a matching
 `version` property. See additional tips for [releasing npm
 packages](#publishing-to-npm).
 
+**Important Note2:** If you are releasing the `openwhisk` or `openwhisk-deploy-kube` repositories,
```
Comment (rabbah) [inline docs/release_instructions.md:177]:
```suggestion
**Important Note 2:** If you are releasing the `openwhisk` or `openwhisk-deploy-kube` repositories, 
```

**Review by rabbah — APPROVED** [review]

**Review by dgrove-oss — APPROVED** [review]
Thanks @style95

**Pull request merged**